### ISSUE TITLE
findByCityAndState should return multiple locations

### ISSUE THREAD

**Issue by callmewa** [sjlu/cities#11]
For cities with more than 1 zipcode it should return all of those locations

**Comment by sjlu** [comment]
Yeah, overlooked this one. I'll have to do a major version release to support this as the output of this function will drastically change.

Also open to PRs if you're willing to add it in.

**Comment by callmewa** [comment]
Zipcodes https://github.com/davglass/zipcodes currents provides this functionality.  I think the code is just a drop in
